bundles/org.openhab.binding.fenecon/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="fenecon"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="fenecon">
+
+		<label>FENECON Home</label>
+		<description>Thing for FENECON Home Device</description>
+		<category>Solarplant</category>
+
+		<channels>
+			<channel id="state" typeId="state"/>
+			<channel id="lastUpdate" typeId="lastUpdate"/>
+			<channel id="essSoc" typeId="essSoc"/>
+			<channel id="chargerPower" typeId="chargerPower"/>
+			<channel id="dischargerPower" typeId="dischargerPower"/>
+			<channel id="emergencyPowerMode" typeId="emergencyPowerMode"/>
+			<channel id="consumptionActivePower" typeId="consumptionActivePower"/>
+			<channel id="productionActivePower" typeId="productionActivePower"/>
+			<channel id="sellToGridPower" typeId="sellToGridPower"/>
+			<channel id="sellToGridEnergy" typeId="sellToGridEnergy"/>
+			<channel id="buyFromGridPower" typeId="buyFromGridPower"/>
+			<channel id="buyFromGridEnergy" typeId="buyFromGridEnergy"/>
+		</channels>
+
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<context>network-address</context>
+				<label>Hostname</label>
+				<description>Hostname or IP address of the FENECON device, e.g. 192.168.1.11</description>
+			</parameter>
+			<parameter name="password" type="text">
+				<context>password</context>
+				<label>Password</label>
+				<default>user</default>
+				<description>Password to access the device. The password for guest access is set by default.</description>
+				<advanced>true</advanced>
```

Comment:
Double check: guest password that is provided by default should give enough autorisations for the binding to work normally? If not it should not be marked as advanced.

Reply:
Yes, the guest password is enough, so it is specified in the [official documentation](https://docs.fenecon.de/_/de/fems/fems-app/OEM_App_REST_JSON.html)